mgmt/rest/client/client.go

```diff
@@ -104,12 +104,14 @@ func Username(u string) metaOp {
 
 var (
 	secureTransport = &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: false},
-		IdleConnTimeout: time.Second,
+		TLSClientConfig:       &tls.Config{InsecureSkipVerify: false},
+		IdleConnTimeout:       time.Second,
+		ResponseHeaderTimeout: 10 * time.Second,
```

Comment:
Does it make sense to add this as a const somewhere so that it can be edited in one place or potentially modified via config in the future?